source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1266,6 +1267,28 @@ message TransformationPermutePhiOperands {
 
 }
 
+message TransformationPropagateInstructionDown {
+
+  // Propagates an instruction from |block_id| into its successors.
+  // Concretely, the transformation clones the propagated instruction
+  // into some of the predecessors of |block_id| and removes the original
+  // instruction. Additionally, OpPhi instruction may be added to make sure
+  // that the transformation can be applied in various scenarios.
+
+  // Id of the block to propagate an instruction from.
+  uint32 block_id = 1;
+
+  // A fresh id for an OpPhi instruction.
```

Comment:
I would reiterate that this may not always be needed.  Or, rather, explain when it is needed: presumably when the instruction is propagated into divergent blocks, and where its original result id is used after those blocks converge?

(BTW: if that is indeed the use case, couldn't the original result id be used for the OpPhi?  Is a fresh id really required?)

Reply:
The fresh id is not required when OpPhi is created. However, I thought it would be more consistent to always use fresh ids since the original id might not always be preserved either (e.g. when OpPhi is not created).

Reply:
Fixed comments in 69b41e9.